### ISSUE TITLE
How to rotate twice like R2 or U2

### ISSUE THREAD

**Issue by nwtgck** [timjb/diagrams-rubiks-cube#5]
Thank you very much for your great project!

 I have a question about `Diagrams.RubiksCube.drawMoves`. Are there way to  rendar 180°-rotate like R2 or U2 by using `drawMoves`?

**Comment by timjb** [comment]
No, drawing 180°-rotations is not currently implemented. I agree that this would be a good feature and I would be happy to accept a pull request implementing this!

**Comment by nwtgck** [comment]
Thank you for your reply. Unfortunately, I'm not familiar with `diagrams-lib` and `lens`. So I'll wait for other pull request!
